kotlin/CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 Change Log
 ==========
 
+## Version 0.12.0
+
+_2019-4-10_
+
+ * Use kotlinx-coroutines-android and Dispatchers.Main instead of Rx's Android scheduler. ([#252](https://github.com/square/workflow/issues/254))
```

Comment:
@rjrjr @zach-klippenstein I tried my best to put these in, based on what the commits had been since 0.11.0 - please let me know if I'm missing something/should amend/etc

Reply:
Wrong link.

Reply:
Whoops, updated. :shame: